mitodl/micromasters
# Allow blank edx_key

#### What are the relevant tickets?
N/A

#### What's this PR do?
Permits a blank edx_key in django admin so courses can be saved without filling it out

#### How should this be manually tested?
Go to edit a course in django admin, change an unrelated field and leave edx_key blank, ensure the page saves.
